### PR TITLE
explorer: add steps to ticket price chart

### LINKED ISSUE #991: Ticket price chart must have steps
Linear interpolation on ticket price chart is misleading: ticket price does not linearly change between adjustments. https://dcred.eu/posStats shows accurate steps.

### PR DESCRIPTION
**Pull request by buck54321**
Resolves #991

### CODE REVIEW

File: public/js/controllers/charts_controller.js [hunk 1]
```diff
@@ -87,12 +87,15 @@ function nightModeOptions (nightModeOn) {
...
-    return [
-      new Date(n),
-      gData.valuef[i]
-    ]
-  })
+  var data = []
+  var lastIdx = gData.time.length - 1
+  for (let i = 0; i < lastIdx; i++) {
+    data.push([new Date(gData.time[i]), gData.valuef[i]])
+    data.push([new Date(gData.time[i + 1]), gData.valuef[i]])
```
Comment (chappjc) [inline public/js/controllers/charts_controller.js]: It's somewhat unfortunate that this doubles the data length.  I was hoping the graph lib would have a plot option like "stairs" or something.

**Comment by chappjc** [comment]
Looks good:

image

File: public/js/controllers/charts_controller.js (hunk 1 above)
Comment (buck54321) [reply public/js/controllers/charts_controller.js]: Actually it does. I didn't see it before, but dygraphs has a `stepPlot` options. I'll fix it now.

Comment (chappjc) [reply public/js/controllers/charts_controller.js]: Excellent!  Thanks.

**Comment by chappjc** [comment]
Still looking good with the `stepPlot` option on.

**Pull request merged**